Algorithm.Framework/Alphas/HistoricalReturnsAlphaModel.py

```diff
@@ -0,0 +1,80 @@
+﻿from clr import AddReference
+AddReference("QuantConnect.Algorithm.Framework")
+AddReference("QuantConnect.Indicators")
+AddReference("QuantConnect.Common")
+
+from QuantConnect import *
+from QuantConnect.Indicators import *
+from QuantConnect.Algorithm.Framework.Alphas import *
+from datetime import timedelta
+
+class HistoricalReturnsAlphaModel:
+    '''Uses Historical returns to create insights.'''
+
+    def __init__(self, *args, **kwargs):
+        '''Initializes a new default instance of the HistoricalReturnsAlphaModel class.
+        Args:
+            resolution: The resolution of historical data'''
+        self.resolution = kwargs['resolution'] if 'resolution' in kwargs else Resolution.Daily
```

Comment:
The resolution should be set by the data coming into the method. Ideally this class should be agnostic

Reply:
We subscribe to intraday data, but Portfolio rebalance is normally done at lower resolution.